### PR TITLE
Several suggestions on non-dynamem codes

### PR DESCRIPTION
**Pull request by hello-peiqi**
## Description

Provide a brief summary of the changes introduced in this pull request.

## Checklist

- \[ \] I have performed a self-review of my code
- \[ \] If it is a core feature, I have added thorough tests
- \[ \] I have added documentation for the changes
- \[ \] I have updated the README file if necessary
- \[ \] I have run on hardware if necessary

## Screenshots (if applicable)

Add any relevant screenshots or screen recordings to help reviewers understand the changes.

## Additional context

Add any other context or information about the pull request here.

### CODE REVIEW

File: src/stretch/config/default_planner.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ detection:
   # module: "yolo_world"
   yolo_world_model_size: "l"   # Choose from ["s", "m", "l", "x"]
   yolo_confidence_threshold: 0.1
```
Comment (hello-peiqi) [inline src/stretch/config/default_planner.yaml:52]: Less bounding boxes, faster speed.

File: src/stretch/agent/task/pickup/pickup_task.py [hunk 2]
```diff
@@ -204,7 +204,11 @@ def get_one_shot_task(self, add_rotate: bool = False, matching: str = "feature")
         task.add_operation(go_to_receptacle)
         task.add_operation(place_object_on_receptacle)
 
-        task.connect_on_success(go_to_navigation_mode.name, search_for_receptacle.name)
+        if not add_rotate:
```
Comment (hello-peiqi) [inline src/stretch/agent/task/pickup/pickup_task.py:207]: bugs in add_rotate

File: src/stretch/agent/robot_agent.py [hunk 3]
```diff
@@ -1361,7 +1361,15 @@ def move_to_nav_posture(self):
         """Move the robot to navigation posture."""
         self.robot.move_to_nav_posture()
 
-    def print_found_classes(self, goal: Optional[str] = None):
+    def is_match_by_feature(self, instance, goal):
```
Comment (hello-peiqi) [inline src/stretch/agent/robot_agent.py:1364]: I assume you guys pass goal in for some purpose right? Maybe just test what instances are detected as goal

File: src/stretch/agent/operations/grasp_object.py [hunk 4]
```diff
@@ -918,9 +918,16 @@ def run(self) -> None:
 
         # Compute the angles necessary
         if self.use_pitch_from_vertical:
-            head_pos = obs.camera_pose[:3, 3]
-            dy = np.abs(head_pos[1] - relative_object_xyz[1])
-            dz = np.abs(head_pos[2] - relative_object_xyz[2])
+            # head_pos = obs.camera_pose[:3, 3]
```
Comment (hello-peiqi) [inline src/stretch/agent/operations/grasp_object.py:921]: Why do we care about head pos when we use visual servo manipulation?

Comment (hello-cpaxton) [reply src/stretch/agent/operations/grasp_object.py:921]: its for computing angle but you're probably right

**Review by hello-cpaxton — APPROVED** [review]
LGTM; tested on stretch 3005

**Pull request merged**